documentation/cobigen-cli_debugging.asciidoc

```diff
@@ -35,4 +42,5 @@ To debug system tests for the CLI,...
 
 3. Repeat step 2 for "cg.bat".
 
-4. Rebuild the CLI by running "mvn clean install -f cobigen-cli/ -DskipTests" in your favorite shell so the files are included in the jar.
+4. If you have changed files in core, rebuild the core by running "mvn clean install -f cobigen/ -DskipTests". +
+Rebuild the CLI by running "mvn clean install -f cobigen-cli/ -DskipTests" in your favorite shell so the files are included in the jar.
```

Comment:
Now we can also use `./build.sh -dpCcore,cli`

Reply:
Resolved.